README.md

```diff
@@ -116,6 +116,13 @@ Please refer to your proxy server's (nginx, apache httpd, etc.) documentation fo
 ## Interacting with the API endpoints
 The design of this API uses JWTs to contain authorization and identity. The Auth endpoint must be invoked first in order to authenticate and receive a JWT. This token will then need to be included in the Authorization header of each subsequent request. This API follows a [HATEOAS](https://en.wikipedia.org/wiki/HATEOAS#:~:text=Hypermedia%20as%20the%20Engine%20of,provide%20information%20dynamically%20through%20hypermedia.) design which provides not only the requested information but possible next operations on that information.
 
+## Error messages
+Unless otherwise specified, successful operations on this endpoint should return with an empty body. If some error is reported, it will follow a standard format. An example is:
```

Comment:
Which endpoint is "this" referring to? Or is it all of them?

Reply:
This applies to all endpoints.

Reply:
@phdavis1027 Let's replace the example with the structure. Showing the structure and the types of each property is more important imo.
```javascript
{
    "error_code": integer,
    "error_message": string
}
```

Reply:
Okay, perhaps the wording should be "...successful operations on all endpoints should return..." or perhaps just "...successful operations should return..."

Reply:
This only applies to the error case too. Success does not always result in a JSON response.

Reply:
True, this was pretty imprecise wording on my part.